src/txn/txn_rollback_to_stable.c

```diff
@@ -718,11 +724,16 @@ __rollback_abort_ondisk_kv(WT_SESSION_IMPL *session, WT_REF *ref, WT_ROW *rip, u
       !__rollback_txn_visible_id(session, vpack->tw.start_txn) ||
       (!WT_TIME_WINDOW_HAS_STOP(&vpack->tw) && prepared)) {
         __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
-          "on-disk update aborted with start durable timestamp: %s, commit timestamp: %s, "
-          "prepared: %s, stable timestamp: %s and txnid : %" PRIu64,
-          __wt_timestamp_to_string(vpack->tw.durable_start_ts, ts_string[0]),
-          __wt_timestamp_to_string(vpack->tw.start_ts, ts_string[1]), prepared ? "true" : "false",
-          __wt_timestamp_to_string(rollback_timestamp, ts_string[2]), vpack->tw.start_txn);
+          "on-disk update with commit timestamp %s aborted because start durable timestamp (%s) > "
+          "stable timestamp "
+          "(%s): %s, or txnid (%" PRIu64
+          ") is not visible: %s, or tw has not stop and is prepared: %s",
+          __wt_timestamp_to_string(vpack->tw.start_ts, ts_string[0]),
+          __wt_timestamp_to_string(vpack->tw.durable_start_ts, ts_string[1]),
```

Comment:
Again, these two `wt_timestamp_to_string` lines seem out of order. The format says durable > stable, but these lines are start_ts then durable_start_ts. Oh  wait, the `start_ts` goes with the `commit timestamp`.

I suggest breaking this one verbose statement into several. The format has so many parts and the various conditions to apply to the format make this very hard to read. Changing it correctly could be challenging.